.github/workflows/release.yaml

```diff
@@ -11,15 +11,12 @@ jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
-
-      - name: Checkout
-        uses: actions/checkout@v1
-
       - name: Set up Go
```

Comment:
This step name should be updated to reflect the changes on go.yml.

Reply:
I updated the name.